frontend/src/views/CellGuide/components/common/OntologyDagView/index.tsx

```diff
@@ -551,7 +557,7 @@ export default function OntologyDagView({
           </TableUnavailableHeader>
           <TableUnavailableDescription>
             The cell ontology visualization for this cell type is currently not
-            available as it is not a descendant of &quot;animal cell&quot;.
+            available.
```

Comment:
When are we going to run into this now?

Reply:
if i'm following the ternary correctly, i believe when `data && initialTreeState` is false, so when we're unable to pull the data for whatever reason.

Reply:
yeah, i understand in theory, but should we add a test to make sure this message is never seen?